src/tutorials/using-a-pathogen-repo/running-an-ingest-workflow.rst

```diff
@@ -0,0 +1,272 @@
+==========================
+Running an ingest workflow
+==========================
+
+This tutorial uses the :term:`Nextstrain CLI` to help you get started running :term:`ingest workflows<ingest workflow>`.
+Ingest workflows download public data from NCBI and output :term:`ingest datasets <ingest dataset>`, which include curated
+metadata and sequences that can be used as input for :term:`phylogenetic <phylogenetic workflow>` or :term:`Nextclade workflows <Nextclade workflow>`.
+
+.. note::
+  You only need to run an ingest workflow if you do not want to use the data files already publicly hosted by Nextstrain.
+  Individual pathogen repositories include documentation that links to their data files.
+
+In this tutorial, you will run the ingest workflow of our `Zika repository <https://github.com/nextstrain/zika>`_ and view outputs on your computer.
+You will have a basic understanding of how to run ingest workflows for other pathogens and a foundation for understanding how to customize ingest workflows.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+=============
+
+1. :doc:`Install Nextstrain </install>`. These instructions will install all of the software you need to complete this tutorial.
+
+Download the Zika repository
+============================
+
+All pathogen ingest workflows are stored in :term:`pathogen repositories<pathogen repository>` (version-controlled folders) to track changes over time.
+Download the `Zika repository <https://github.com/nextstrain/zika>`_.
+
+.. code-block::
+
+    $ git clone https://github.com/nextstrain/zika
+    Cloning into 'zika'...
+    [...more output...]
+
+When it's done, you'll have a new directory called ``zika/``.
+
+Run the default workflow
+========================
+
+The zika :term:`ingest workflow` uses the `NCBI Datasets CLI tools <https://www.ncbi.nlm.nih.gov/datasets/docs/v2/reference-docs/command-line/>`_
+to download public data and uses a combination of :doc:`augur curate <augur:usage/cli/curate/index>` and other data manipulation tools to curate
+the downloaded data into a format suitable for :term:`phylogenetic workflows <phylogenetic workflow>`.
+
+1. Change directory to the Zika pathogen repository downloaded in the previous step
+
+.. code-block::
+
+    $ cd zika
+
+2. Run the default ingest workflow with the :term:`Nextstrain CLI`.
+
+.. code-block::
+
+    $ nextstrain build ingest
+    Using profile profiles/default and workflow specific profile profiles/default for setting default command line arguments.
+    Building DAG of jobs…
+    [...a lot of output...]
```

Comment:
All of these new code-blocks which show an example shell session with a prompt and command and optionally output should be `.. code-block:: console` to properly handle the various bits inside with highlighting and the copy/paste button.

Reply:
That's good to know! I just tried updating to `.. code-block::console` locally, I see a difference in syntax highlighting but the copy/paste behavior is still the same where it only copies the command.

Reply:
Adding missing code-block languages in https://github.com/nextstrain/docs.nextstrain.org/pull/197

Reply:
> …but the copy/paste behavior is still the same where it only copies the command.

Ah! I thought we were using the [syntax-aware prompt/output exclusion method](https://sphinx-copybutton.readthedocs.io/en/latest/use.html#automatic-exclusion-of-prompts-from-the-copies) of sphinx-copybutton, but [we're using](https://github.com/nextstrain/sphinx-theme/blob/d1ccdba97b571d8655ab48b63fbe9decf9d96071/lib/nextstrain/sphinx/theme/__init__.py#L21-L22) the [pattern-matching exclusion method](https://sphinx-copybutton.readthedocs.io/en/latest/use.html#strip-and-configure-input-prompts-for-code-cells).

All good then.